src/set_theory/game/basic.lean

```diff
@@ -133,16 +139,93 @@ namespace pgame
 
 @[simp] lemma quot_sub (a b : pgame) : ⟦a - b⟧ = ⟦a⟧ - ⟦b⟧ := rfl
 
+/-- The lemma `pgame.equiv_of_mk_equiv` written in terms of quotients. -/
 theorem quot_eq_of_mk_quot_eq {x y : pgame}
   (L : x.left_moves ≃ y.left_moves) (R : x.right_moves ≃ y.right_moves)
-  (hl : ∀ (i : x.left_moves), ⟦x.move_left i⟧ = ⟦y.move_left (L i)⟧)
-  (hr : ∀ (j : y.right_moves), ⟦x.move_right (R.symm j)⟧ = ⟦y.move_right j⟧) :
+  (hl : ∀ i, ⟦x.move_left i⟧ = ⟦y.move_left (L i)⟧)
+  (hr : ∀ j, ⟦x.move_right (R.symm j)⟧ = ⟦y.move_right j⟧) :
   ⟦x⟧ = ⟦y⟧ :=
-begin
-  simp only [quotient.eq] at hl hr,
-  apply quotient.sound,
-  apply equiv_of_mk_equiv L R hl hr,
-end
+by { simp_rw quotient.eq at hl hr ⊢, exact equiv_of_mk_equiv L R hl hr }
+
+/-! We prove various inequalities on pre-games taking advantage of the extra structure on games. -/
+
+theorem neg_le {x y : pgame} : -x ≤ y ↔ -y ≤ x :=
+@neg_le game _ _ _ _ ⟦x⟧ ⟦y⟧
+
+theorem neg_lt {x y : pgame} : -x < y ↔ -y < x :=
+@neg_lt game _ _ _ _ ⟦x⟧ ⟦y⟧
+
+theorem le_neg {x y : pgame} : x ≤ -y ↔ y ≤ -x :=
+@le_neg game _ _ _ _ ⟦x⟧ ⟦y⟧
+
+theorem lt_neg {x y : pgame} : x < -y ↔ y < -x :=
+@lt_neg game _ _ _ _ ⟦x⟧ ⟦y⟧
+
+theorem neg_lf {x y : pgame} : -x ⧏ y ↔ -y ⧏ x :=
+by rw [←le_iff_le_iff_lf_iff_lf, le_neg]
+
+theorem lf_neg {x y : pgame} : x ⧏ -y ↔ y ⧏ -x :=
+by rw [←le_iff_le_iff_lf_iff_lf, neg_le]
+
+theorem equiv_neg_iff_equiv_neg {x y : pgame} : x ≈ -y ↔ y ≈ -x :=
+by simp only [equiv, le_neg, neg_le, and_comm]
+
+theorem neg_equiv_iff_neg_equiv {x y : pgame} : -x ≈ y ↔ -y ≈ x :=
+by simp only [equiv, le_neg, neg_le, and_comm]
+
+theorem fuzzy_neg_iff_fuzzy_neg {x y : pgame} : x ∥ -y ↔ y ∥ -x :=
+by simp only [fuzzy, lf_neg, neg_lf, and_comm]
+
+theorem neg_fuzzy_iff_neg_fuzzy {x y : pgame} : -x ∥ y ↔ -y ∥ x :=
+by simp only [fuzzy, lf_neg, neg_lf, and_comm]
+
+@[simp] theorem neg_le_zero_iff {x : pgame} : -x ≤ 0 ↔ 0 ≤ x :=
+@neg_nonpos game _ _ _ ⟦x⟧
+
+@[simp] theorem zero_le_neg_iff {x : pgame} : 0 ≤ -x ↔ x ≤ 0 :=
+@neg_nonneg game _ _ _ ⟦x⟧
+
+@[simp] theorem neg_lt_zero_iff {x : pgame} : -x < 0 ↔ 0 < x :=
+@neg_lt_zero game _ _ _ ⟦x⟧
+
+@[simp] theorem zero_lt_neg_iff {x : pgame} : 0 < -x ↔ x < 0 :=
+@neg_pos game _ _ _ ⟦x⟧
+
+@[simp] theorem neg_lf_zero_iff {x : pgame} : -x ⧏ 0 ↔ 0 ⧏ x :=
+by rw [neg_lf, pgame.neg_zero]
+
+@[simp] theorem zero_lf_neg_iff {x : pgame} : 0 ⧏ -x ↔ x ⧏ 0 :=
```

Comment:
I think `_iff` can be removed from the name for these lemmas .

Reply:
Sure thing.